user-interface/src/main/java/life/qbic/datamanager/views/projects/project/measurements/download/NGSMeasurementContentProvider.java

```diff
@@ -180,23 +172,47 @@ public byte[] getContent() {
     ByteArrayOutputStream byteArrayOutputStream;
 
     try (Workbook workbook = new XSSFWorkbook()) {
-      Sheet sheet = workbook.createSheet("NGS Measurement Metadata");
-
-      Row header = sheet.createRow(0);
       defineReadOnlyHeaderStyle(workbook);
       defineReadOnlyCellStyle(workbook);
       defineBoldStyle(workbook);
-      formatHeader(header);
 
-      int rowCounter = 1;
+      Sheet sheet = workbook.createSheet("NGS Measurement Metadata");
+
+      Row header = getOrCreateRow(sheet, 0);
+      for (NGSMeasurementColumns value : NGSMeasurementColumns.values()) {
+        var cell = getOrCreateCell(header, value.columnIndex());
+        cell.setCellValue(value.headerName());
+        setHeaderStyle(cell, value.readOnly());
+      }
 
-      for (NGSMeasurementEntry ngsMeasurementEntry : measurements) {
-        Row entry = sheet.createRow(rowCounter);
-        createMeasurementEntry(ngsMeasurementEntry, entry);
-        rowCounter++;
+      var startIndex = 1; // start in row number 2 with index 1 as the header row has number 1 index 0
+      int rowIndex = startIndex;
+      for (NGSMeasurementEntry measurement : measurements) {
+        Row row = getOrCreateRow(sheet, rowIndex);
+        writeMeasurementIntoRow(measurement, row);
+        rowIndex++;
       }
 
+      var generatedRowCount = rowIndex - startIndex;
+      assert generatedRowCount == measurements.size() : "all measurements have a corresponding row";
+
+      // make sure to create the visible sheet first
+      Sheet hiddenSheet = workbook.createSheet("hidden");
+      Name sequencingReadTypeArea = createOptionArea(hiddenSheet,
+          "Sequencing read type", SequencingReadType.getOptions());
+
+      XLSXTemplateHelper.addDataValidation(sheet,
+          NGSMeasurementColumns.SEQUENCING_READ_TYPE.columnIndex(),
+          startIndex,
+          NGSMeasurementColumns.SEQUENCING_READ_TYPE.columnIndex(),
+          DEFAULT_GENERATED_ROW_COUNT - 1,
+          sequencingReadTypeArea);
+
       setAutoWidth(sheet);
+      workbook.setActiveSheet(0);
+
+      lockSheet(hiddenSheet);
+      hideSheet(workbook, hiddenSheet);
```

Comment:
I'm confused, why do we need a seperate hidden sheet now?

Reply:
The selection of possible values is stored in this hidden sheet. The values are included in a named area in this hidden sheet. This named area is used as formula for the dropdown.https://thecodeshewrites.com/2020/08/11/apache-poi-excel-java-dropdown-list-dependent/